### DISCUSSION TITLE
Inline edit of a row in table component?

### DISCUSSION THREAD

**Discussion by YunusMechery** [sqlpage/SQLPage#568] [Q&A] [answered]
Is there a way to edit a single field of a row in a table without loading another update form, like task status to completed? Something like the example at the bottom of this page https://htmx.org/examples/edit-row/

**Comment by lovasoa** [comment, marked as answer]
Hi Yunus !
  We are currently working on a premium *spreadsheet* component that will allow you to implement editable tables (and more)

![image](https://github.com/user-attachments/assets/732bbd54-d7b5-4d46-b321-f6dfa866e409)


Don't hesitate to ask @alexisrc1 for more info ;)

edit: the spreadsheet component is now live on https://github.com/sqlpage/sqlpage-spreadsheet/

**Comment by YunusMechery** [comment]
@alexisrc1 I came to SQLPage from Google Sheets. Are you taking me back to something similar? :(

**Comment by ToniJ123** [comment]
Could this be somehow achieved still, so table component's row's would have dropdown menu option for some column example and editable text field for some column, if choosen so?
It would be great for some use cases, where this spreadsheed is too heavy or complex for just editing single column and wanting to see it via table component. 👍
